sympy/core/basic.py

```diff
@@ -1525,6 +1525,52 @@ def match(self, pattern, old=False):
         pattern = sympify(pattern)
         return pattern.matches(self, old=old)
 
+    def partition(self, var):
```

Comment:
Thanks! This function is not for end-users.

I have renamed the function to `_partiton_args()`